docs/arcaflow/workflows/expressions.md

```diff
@@ -15,16 +15,18 @@ This page explains the language elements of expressions.
 
 Literals represent values described in an expression, as opposed to values referenced from other sources.
 
-### String
+### String values
 
-string literals start and end with a matched pair of either single quotes `'` or double quotes `"`, and have zero or more characters between the quotes.
+Normal string literals start and end with a matched pair of either single quotes `'` or double quotes `"` and have zero or more characters between the quotes.
 
 Strings may have escaped values. The most important ones are for backslashes (`\\` for `\`) or for embedded newlines `\n`.
 
+You do not need to escape double quotes in a single-quote string, or single-quotes in a double quote string. However, 
```

Comment:
In addition to dropping the `However,`, you should omit the other comma, as well.  And, consider replacing "or" with "nor".